rfcs/0004-leader-rotation.md

```diff
@@ -1,29 +1,23 @@
 # Leader Rotation
 
-The goal of this RFC is to define how leaders rotate in taking their role.
+The goal of this RFC is to define how fullnodes rotate in taking the leader role.
 
 ## Leader Schedule Generation
 
 Leader schedule is decided via a predefined seed.  The process is as follows:
 
-1. Periodically at a specific `PoH tick count` use the tick count (simple monotonically increasing counter) as a seed to a stable pseudo-random algorithm.
-2. At that height, example the bank for all the currently staked accounts and their assigned leader identities and weights.
+1. Periodically use the PoH tick height (a monotonically increasing counter) to seed a stable pseudo-random algorithm.
+2. At that height, sample the bank for all the currently staked accounts, their assigned leader identities and weights.
```

Comment:
We also take into account an "active set", which includes all fullnodes that have voted within the last "n" ticks.